deployment/src/main/java/io/quarkiverse/loggingmanager/quarkus/logging/manager/deployment/LoggingManagerBuildItem.java

```diff
@@ -1,4 +1,4 @@
-package io.quarkiverse.loggingui.quarkus.logging.ui.deployment;
+package io.quarkiverse.loggingmanager.quarkus.logging.manager.deployment;
```

Comment:
Do we really need this part in the package name ? How about just `io.quarkiverse.loggingmanager.deployment` ?

Reply:
Good point. I don't know why this was this way. But I will change.